examples/ExampleDenseArray.cs

```diff
@@ -137,6 +137,10 @@ private static TileDB.QueryStatus ReadDenseSimpleArray()
             TileDB.QueryStatus status = query.submit();
             array.close();
 
+            for (int i = 0; i < data.Count; ++i)
+            {
+                System.Console.WriteLine("data at {0}:{1}", i, data[i]);
```

Comment:
please add space after `:` here and below

Reply:
also, printing the index `i` here doesn't really make sense because the array is 2D. so IMO we should drop that and just print the result vector.

Reply:
changed to System.Console.WriteLine(String.Join(" ", data));